packages/go/safety/properties_test.go

```
@@ -0,0 +1,171 @@
+	"testing/quick"
+)
+
+// PropertyDeterministic P1：同输入两次 Classify/PreSpeak 决策一致（Sev/SpokenText/
+// Intercepted/Notify 存在性——决策面无随机漂移）。
+func PropertyDeterministic(text string) bool {
+	e, err := NewEngine(DefaultConfig())
+	if err != nil {
+		return false
+	}
+	d1, d2 := e.PreSpeak(text), e.PreSpeak(text)
+	return e.Classify(text) == e.Classify(text) &&
```

golangci-lint (2.12.2)

[error] SA4000: identical expressions on the left and right side of the '==' operator

(staticcheck)